sdk/nodejs/resource.ts

```diff
@@ -480,6 +480,13 @@ export class Output<T> {
 To manipulate the value of this Output, use '.apply' instead.`);
         };
     }
+
+    // We override `toString` to provide more useful guidance when `Output`s are converted to strings, instead of just
+    // printing `[object Object]`.
+    toString() {
+        return "<<< Warning: called 'toString' on an Output, use 'apply' instead: https://pulumi.io/help/outputs >>>";
```

Comment:
I think that this would read better as `<<< Warning: call 'apply' instead of 'toString' on an Output`: https://pulumi.io/help/outputs >>>`.

It might also be nice to have a deep link to specific help for this scenario that suggests e.g. `.apply(v => v.toString())`